example/shared/src/commonMain/kotlin/viewmodel/monitor/ServiceMonitorViewModel.kt

```diff
@@ -0,0 +1,57 @@
+/*
+ Copyright 2021 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.example.shared.viewmodel.monitor
+
+import com.splendo.kaluga.architecture.observable.toInitializedObservable
+import com.splendo.kaluga.architecture.viewmodel.BaseViewModel
+import com.splendo.kaluga.bluetooth.BluetoothMonitor
+import com.splendo.kaluga.base.monitor.ServiceMonitorStateRepo
+import com.splendo.kaluga.base.monitor.ServiceMonitorState
+import com.splendo.kaluga.resources.localized
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.launch
+
+class ServiceMonitorViewModel(bltMonitor: BluetoothMonitor) : BaseViewModel() {
```

Comment:
Weird prefix, `ble` or just `monitor`?

Reply:
Later I will also add `locationMonitor` so it cannot just be `monitor`. `ble` is low energy, `blt` is just shortage for bluetooth.